Mathlib/Logic/Equiv/LocalEquiv.lean

```diff
@@ -244,17 +244,27 @@ protected theorem surjOn : SurjOn e e.source e.target :=
   e.bijOn.surjOn
 #align local_equiv.surj_on LocalEquiv.surjOn
 
-/-- Associate a `LocalEquiv` to an `Equiv`. -/
-@[simps (config := mfld_cfg)]
-def _root_.Equiv.toLocalEquiv (e : α ≃ β) : LocalEquiv α β where
+/-- Interpret an `Equiv` as a `LocalEquiv` by restricting it to `s` in the domain
+and to `t` in the codomain. -/
+@[simps (config := .asFn)]
+def _root_.Equiv.toLocalEquivOfImageEq (e : α ≃ β) (s : Set α) (t : Set β) (h : e '' s = t) :
```

Comment:
I'm still learning our new naming conventions but I believe this should be:
```suggestion
def _root_.Equiv.toLocalEquiv_of_image_eq (e : α ≃ β) (s : Set α) (t : Set β) (h : e '' s = t) :
```

Reply:
We may want to change the name convention but currently we have no underscores in names of `def`s.